x/ibc-transfer/types/packet_test.go

```diff
@@ -6,34 +6,31 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+var (
+	denom  = "atom"
```

Comment:
```suggestion
	denom  = "transfer/gaiachannel/atom"
```

nit: A slightly stronger test